validation_config.toml

```diff
@@ -27,7 +27,7 @@ select = [
     "BC100", "BC101", "BC102", "BC103", "BC104", "BC105", "BC106", "BC107", "BC108", "BC110", "BC111", "BC112", "BC113", "BC114", "BC115",
     "IN100", "IN101", "IN102", "IN104", "IN106", "IN107", "IN108", "IN109", "IN110", "IN112", "IN113", "IN114", "IN115", "IN117", "IN118", "IN121", "IN122", "IN123", "IN124", "IN125", "IN126", "IN127", "IN130",
     "IN131", "IN134", "IN135", "IN139", "IN141", "IN142", "IN144", "IN145", "IN146", "IN149", "IN151", "IN152", "IN153", "IN154", "IN156", "IN158", "IN159", "IN160",
-    "IN161", "IN162", "IN163",
+    "IN161", "IN162", "IN163", "IN164",
```

Comment:
@yedidyacohenpalo
Are we sure we want to add the new validation in this PR? Didn't we want add it after the Infra release?